src/components/Buttons/SignInWithMatters/index.tsx

```diff
@@ -43,14 +52,25 @@ export const SignInWithMatters = () => {
   }
 
   return (
-    <CardButton
-      title="請登入 Matters 帳戶"
-      leftIcon={<IconUser size="xlM" />}
-      right={loading ? <IconSpinner /> : <IconArrowRight />}
-      htmlHref={env.mattersLoginURL}
-      htmlTarget="_blank"
-      onClick={signIn}
-      disabled={loading || !!error}
-    />
+    <>
+      <CardButton
+        title={
+          locale === Lang.en ? "Sign In with Matters" : "請登入 Matters 帳戶"
```

Comment:
Beryl uses `Matters.news` to specify the web app product.

Reply:
Yes. I think it's fine since the dialog description has already mentioned Matters.news.

Reply:
Got it. 👌🏿